src/Compilers/Core/Portable/Operations/ITupleBinaryOperation.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis.Operations
+{
+    /// <summary>
+    /// Represents a comparison of two operands that returns a bool type.
+    /// <para>
+    /// Current usage:
+    ///  (1) C# tuple binary operator expression.
+    /// </para>
+    /// </summary>
+    /// <remarks>
+    /// This interface is reserved for implementation by its associated APIs. We reserve the right to
+    /// change it in the future.
+    /// </remarks>
+    public interface ITupleBinaryOperation : IOperation
```

Comment:
>ITupleBinaryOperation [](start = 21, length = 21)

Do we really have to add a special node for this? Can we reuse IBinaryOperation?

Reply:
I assume we'll want to expose the tree of element-wise operators at some point.

Reply:
Filed [https://github.com/dotnet/roslyn/issues/25435](https://github.com/dotnet/roslyn/issues/25435) for follow-up (consider adding more information to the ITupleBinaryOperation node)

---
In reply to: [173959373](https://github.com/dotnet/roslyn/pull/25374#discussion_r173959373) [](ancestors = 173959373)